app/templates/administrative-units/administrative-unit/sites/site/index.hbs

```diff
@@ -0,0 +1,100 @@
+<div class="au-o-box">
+  <AuToolbar>
+    <AuToolbarGroup>
+      <AuHeading>
+        Vestiging: {{@model.sites.address.fullAddress}}
+      </AuHeading>
+    </AuToolbarGroup>
+    <AuToolbarGroup>`
+      {{! TODO: Replace this with a link once the "new" route is created }}
+      <AuButton @icon="add" @iconAlignment="left">
+        Nieuw
+      </AuButton>
+    </AuToolbarGroup>
+  </AuToolbar>
+  <AuHeading @level="2" @skin="3" class="au-u-margin-top-tiny">
+    {{@model.administrativeUnit.name}}
+  </AuHeading>
+</div>
+
+<div class="au-o-box">
+  <AuToolbar class="au-u-margin-bottom-small">
+    <AuToolbarGroup>
+      <AuHeading @level="4" @skin="5">Kerngegevens</AuHeading>
+    </AuToolbarGroup>
+  </AuToolbar>
+  <AuCard @standOut={{true}} as |c|>
+    <c.content>
+      <div class="au-flow au-o-flow--large">
+        <div class="au-o-grid">
+          <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+            <div class="au-o-grid">
+              <div class="au-o-grid__item au-u-1-4">
+                <p><strong>Hoofgebouw</strong></p>
+              </div>
+              <div class="au-o-grid__item au-u-3-4">
+                <p>{{if (eq @model.sites.id @model.primarySite.id) "Ja" "Nee"}}</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </c.content>
+  </AuCard>
+</div>
+
+<div class="au-o-box">
+  <AuToolbar class="au-u-margin-bottom-small">
+    <AuToolbarGroup>
+      <AuHeading @level="4" @skin="5">Contactgegevens</AuHeading>
+    </AuToolbarGroup>
+  </AuToolbar>
+  <AuCard @standOut={{true}} as |c|>
+    <c.content>
+      <div class="au-flow au-o-flow--large">
+        <div class="au-o-grid">
+          <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+            <div class="au-o-grid">
+              <div class="au-o-grid__item au-u-1-4">
+                <p><strong>Adres</strong></p>
+              </div>
+              <div class="au-o-grid__item au-u-3-4">
+                <p>{{@model.sites.address.fullAddress}}</p>
+              </div>
+            </div>
+          </div>
+          <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+            <div class="au-o-grid">
+              <div class="au-o-grid__item au-u-1-4">
+                <p><strong>Provincie</strong></p>
+              </div>
+              <div class="au-o-grid__item au-u-3-4">
+                <p>{{@model.sites.address.province}}</p>
+              </div>
+            </div>
+          </div>
+          <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+            <div class="au-o-grid">
+              <div class="au-o-grid__item au-u-1-4">
+                <p><strong>Telefoonnummer</strong></p>
+              </div>
+              <div class="au-o-grid__item au-u-3-4">
+                <p>{{@model.sites.contacts.firstObject.telephone}}</p>
```

Comment:
I think most of these values can be empty, so I would wrap them in a check to not show the label if there is no value.